lib/mongoid/contextual/mongo.rb

```diff
@@ -158,12 +158,27 @@ def each(&block)
       # @example Do any documents exist for the context.
       #   context.exists?
       #
+      # @example Do any documents exist for given _id.
+      #   context.exists?(BSON::ObjectId(...))
+      #
+      # @example Do any documents exist for given conditions.
+      #   context.exists?(name: "...")
+      #
       # @note We don't use count here since Mongo does not use counted
       #   b-tree indexes.
       #
+      # @param [ Hash | Object | false ] id_or_conditions an _id to
+      #   search for, a hash of conditions, or false.
+      #
       # @return [ true | false ] If the count is more than zero.
-      def exists?
-        !!(view.projection(_id: 1).limit(1).first)
+      #   Always false if passed false.
+      def exists?(id_or_conditions = nil)
+        case id_or_conditions
+        when nil then !!(view.projection(_id: 1).limit(1).first)
+        when false then false
```

Comment:
Whats the reason `false` returns false? Is this something from Rails?

Reply:
Yeah check out the discussion above.... Considering removing this